usecases/webhook_events_usecase.go

```diff
@@ -117,12 +115,6 @@ func (usecase WebhookEventsUsecase) SendWebhookEventAsync(ctx context.Context, w
 	go func() {
 		ctx, cancel := context.WithTimeout(context.WithoutCancel(ctx), ASYNC_WEBHOOKS_SEND_TIMEOUT)
 		defer cancel()
-		tracer := utils.OpenTelemetryTracerFromContext(ctx)
```

Comment:
I'm removing this because, as it's run asynchronously, it sometimes results in super long spans (waiting for the "internal" rate limiting for convoy), which make the spans UI unreadable